.github/pull_request_template.md

```diff
@@ -2,6 +2,10 @@
 
 *Describe the change in this PR*
 
+If this PR changes frontend code, please list the affected feature(s) below:
+
+1. XXXXXXXX
```

Comment:
this information is not valid/necessary for most doc contributors. I suggest not to put it here. We can track this type of information in issue tracker.